### PR TITLE
add result_ttl

### PR DESCRIPTION
**Pull request by ch40s3k**

### CODE REVIEW

File: alamo_worker/cli.py [hunk 1]
```diff
@@ -57,10 +58,17 @@ def parse_config(self):
         self.config = ConfigParser()
         self.config.read(config_file)
 
+    def _get_result_ttl(self):
+        try:
+            return self.config.get('sentinel', 'result_ttl')
```
Comment (pawellesniewski) [inline alamo_worker/cli.py]: sentinel -> worker

**Comment by pawellesniewski** [comment]
🍰

**Comment by h00dy** [comment]
🔥

**Comment by pawellesniewski** [comment]
🍰 🍰

**Comment by h00dy** [comment]
🚒

**Pull request merged**